docs/getting-started/bpfctl-guide.md

```diff
@@ -16,48 +16,28 @@ sudo ./target/debug/bpfctl list
 ```
 
 If run as a systemd service, `bpfctl` will most likely be installed in your $PATH,
-the `bpfd` user and user group were created, so the usergroup `bpfd` will need to be
-added to the desired user.
-Then `sudo` is no longer required.
+and will also require `sudo`.
 Example:
 
 ```console
-sudo usermod -a -G bpfd $USER
-exit
-<LOGIN>
-
-bpfctl list
+sudo bpfctl list
 ```
 
-The examples here use `bpfctl` in place of `sudo ./target/debug/bpfctl` for readability,
+The examples here use `sudo bpfctl` in place of `sudo ./target/debug/bpfctl` for readability,
 use as your system is deployed.
 
-### bpfctl load-from-file With bpfd As A Systemd Service
-
-For security reasons, when `bpfd` is run as a systemd service, all linux capabilities are stripped
-from any spawned threads.
-Therefore, `bpfd` can only access files owned by the `bpfd` user group.
-This includes eBPF object files.
-In the `bpfctl load-from-file` examples below, the files are located in `/run/bpfd/examples/`, which is
-a directory owned by `bpfd`.
-Below is an example of copying local files over for use in this scenario:
-
-```console
-sudo cp /$HOME/src/xdp-tutorial/basic01-xdp-pass/xdp_pass_kern.o /run/bpfd/examples/.
-sudo cp /$HOME/src/net-ebpf-playground/.output/filter.bpf.o /run/bpfd/examples/.
-sudo chown bpfd:bpfd -R /run/bpfd/examples/
-```
-
-This is only needed if `bpfd` is run as a systemd service.
-When `sudo ./scripts/setup.sh install` is run to launch bpfd as a systemd service, all the
-eBPF object files from the `examples/` directory are copied to `/run/bpfd/examples/`.
+eBPF object files used in the examples are taken from the
```

Comment:
Aren't they ALL now defined in `/bpfd/examples/` and `bpfd/integration-test/bpf` now?